notify/webhook/webhook.go

```diff
@@ -73,9 +73,17 @@ type Message struct {
 	GroupKey string `json:"groupKey"`
 }
 
+func truncateAlerts(maxAlerts uint32, alerts []*types.Alert) []*types.Alert {
+	if maxAlerts != 0 && uint32(len(alerts)) > maxAlerts {
```

Comment:
```suggestion
	if maxAlerts != 0 &&len(alerts) > int(maxAlerts) {
```

Reply:
I think this is safer as we know that maxalert will always fit in an int, but len() cound not fit in uint32.

Also, maybe we should use uint() instead of uint32() for simplicity.

Reply:
How about I just use a uint64... then anything will fit in there, and it's not like we need to save space for this one :)

Reply:
uint sounds nicer to me and just shows what we want: a positive int. but it's up to you.

Reply:
FWIW I'm fine with a `uint64`